tools/nl/embeddings/input/undata_ilo/sheets_svs.csv

```diff
@@ -1,86 +1,33 @@
-dc/topic/ILOLUUXLUXNB,Jobs Gap
-dc/topic/ILOLUUXLUXRT,Jobs Gap Rate
-dc/topic/ILOPOP3FORNB,Youth Working-age Population thousands
-dc/topic/ILOPOP3STGNB,Youth Working-age Population thousands
-dc/topic/ILOPOP3TEDNB,"Youth Transited To Stable Employment, Satisfactory Temporary Employment, or Satisfactory Self-employment"
-dc/topic/ILOPOP3WAPNB,Youth Working-age Population
-dc/topic/ILOPSETPSENB,Public Sector Employment
-dc/topic/ILOSDG0111RT,Proportion of Population Below The International Poverty Line SDG Indicator 1.1.1
-dc/topic/ILOSDG0831RT,Proportion of Informal Employment in Total Employment SDG Indicator 8.3.1
-dc/topic/ILOSDG0861RT,"Proportion of Youth aged 15-24 Years Not in Education, Employment or Training SDG Indicator 8.6.1"
-dc/topic/ILOSDGF881RT,Fatal Occupational Injuries Per 100'000 Workers SDG Indicator 8.8.1
-dc/topic/ILOSDGN881RT,Non-fatal Occupational Injuries Per 100'000 Workers SDG Indicator 8.8.1
-dc/topic/ILOSTRDAYSNB,Days Not Worked Due To Strikes And Lockouts
-dc/topic/ILOSTRDAYSRT,Rates of Days Not Worked Due To Strikes And Lockouts
-dc/topic/ILOSTRTSTRNB,Strikes And Lockouts
-dc/topic/ILOSTRWORKNB,Workers Involved in Strikes And Lockouts
-dc/topic/ILOTRU3TRUNB,Youth Time-related Underemployment
-dc/topic/ILOTRU5EMPRT,"Time-related Underemployment Rate, According To Standards Adopted At The 19th International Conference of Labour Statisticians ICLS"
-dc/topic/ILOTRU5TRUNB,"Time-related Underemployment, According To Standards Adopted At The 19th International Conference of Labour Statisticians ICLS"
-dc/topic/ILOUNE2EAPRT,Unemployment Rate ILO Modelled Estimates
-dc/topic/ILOUNE2UNENB,Unemployment ILO Modelled Estimates
-dc/topic/ILOUNE3EAPRT,Youth Unemployment Rate
-dc/topic/ILOUNE3UNENB,Youth Unemployment
-dc/topic/ILOUNE3WAPRT,Youth Unemployment-to-population Ratio
-dc/topic/ILOUNE5EAPRT,"Unemployment Rate, According To Standards Adopted At The 19th International Conference of Labour Statisticians ICLS"
-dc/topic/ILOUNETUNENB,"Unemployment Labour Force Statistics, LFS"
+dc/topic/ILOGED2LFPRT,"Prime-age Labour Force Participation Rate of Couple Households With Children Under Age 6 ILO Modelled Estimates"
+dc/topic/ILOINJDAYSNB,"Days Lost Due To Cases of Occupational Injury With Temporary Incapacity for Work"
+dc/topic/ILOINJFATLNB,"Cases of Fatal Occupational Injury"
+dc/topic/ILOINJFATLRT,"Fatal Occupational Injuries Per 100'000 Workers"
+dc/topic/ILOINJNFTLNB,"Cases of Non-fatal Occupational Injury"
+dc/topic/ILOINJNFTLRT,"Non-fatal Occupational Injuries Per 100'000 Workers"
+dc/topic/ILOLAC4HRLNB,"Mean Nominal Hourly Labour Cost Per Employee"
+dc/topic/ILOLAIINSPNB,"Number of Labour Inspectors"
+dc/topic/ILOLAP2LIDRT,"Labour Income Distribution  ILO Modelled Estimates"
+dc/topic/ILOLUUXLU2RT,"Combined Rate of Time-related Underemployment And Unemployment LU2"
+dc/topic/ILOLUUXLUXNB,"Jobs Gap"
+dc/topic/ILOLUUXLUXRT,"Jobs Gap Rate"
+dc/topic/ILOSTRDAYSNB,"Days Not Worked Due To Strikes And Lockouts"

```

Comment:
Number of Days? Give the next one are very similar, this needs to be distinct enough.

Reply:
Added to alternatives